packages/next/server/load-components.ts

```diff
@@ -94,16 +97,28 @@ export async function loadComponents(
 
   const DocumentMod = await requirePage('/_document', distDir, serverless)
   const AppMod = await requirePage('/_app', distDir, serverless)
-  const ComponentMod = await requirePage(pathname, distDir, serverless)
+  const ComponentMod = await requirePage(
```

Comment:
This would be a good place for a `Promise.all` though...

Reply:
Great catch, will file a separate PR for this!

Reply:
Created #29494.

Reply:
Hmm I just realized that they're just bare `require`s which are not promises...